kork-cloud-config-server/kork-cloud-config-server.gradle

```
@@ -6,6 +6,7 @@ dependencies {
 
   implementation "commons-io:commons-io"
   implementation "org.apache.commons:commons-lang3"
+  implementation project(':kork-config')
```

I tried this locally and it looks like this only needs to be 
```suggestion
  testImplementation project(':kork-config')
```
so DefaultPropertiesBuilder is available to ConfigServerTestApplication.

---

Updated the dependency as suggested.